src/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+
+const productRouter = require("./productRouter");
+router.use("/products", productRouter.router) 
+
+
+module.exports = router;
```

Comment:
```suggestion
const express = require("express");
const router = express.Router();

const userRouter = require("./userRouter");
const productRouter = require("./productRouter");

router.use("/users", userRouter.router);
router.use("/products", productRouter.router) 

module.exports = router;
```
1. 모듈을 불러오는 절차를 가장 최상단에서 진행하시는 것이 좋습니다